paimon-core/src/main/java/org/apache/paimon/partition/actions/PartitionMarkDoneAction.java

```diff
@@ -16,31 +16,38 @@
  * limitations under the License.
  */
 
-package org.apache.paimon.flink.sink.partition;
+package org.apache.paimon.partition.actions;
 
 import org.apache.paimon.CoreOptions;
 import org.apache.paimon.metastore.MetastoreClient;
 import org.apache.paimon.table.FileStoreTable;
+import org.apache.paimon.utils.StringUtils;
 
 import java.io.Closeable;
 import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
 
 import static org.apache.paimon.CoreOptions.METASTORE_PARTITIONED_TABLE;
-import static org.apache.paimon.flink.FlinkConnectorOptions.PARTITION_MARK_DONE_ACTION;
 import static org.apache.paimon.utils.Preconditions.checkArgument;
 import static org.apache.paimon.utils.Preconditions.checkNotNull;
 
 /** Action to mark partitions done. */
 public interface PartitionMarkDoneAction extends Closeable {
 
+    String PARTITION_MARK_DONE_ACTION = "partition.mark-done-action";
+    String DEFAULT_MARK_DONE_ACTION = "success-file";
+
     void markDone(String partition) throws Exception;
 
     static List<PartitionMarkDoneAction> createActions(
             FileStoreTable fileStoreTable, CoreOptions options) {
-        return Arrays.asList(options.toConfiguration().get(PARTITION_MARK_DONE_ACTION).split(","))
-                .stream()
+        String markdoneAction =
+                StringUtils.isBlank((options.toConfiguration().get(PARTITION_MARK_DONE_ACTION)))
```

Comment:
Why we need to handle blank?

Reply:
Because in Flink handle logic would process in default value in PARTITION_MARK_DONE_ACTION conf if it is not set，but the config is in Flink module not in core or spark moudle，so keep the common logic like this way.